tests/js/server/aql/aql-geo.js

```diff
@@ -1045,8 +1045,10 @@ function searchAliasGeoSuite() {
   return suite;
 }
 
+/*
 jsunity.run(arangoSearchGeoSuite);
 jsunity.run(searchAliasGeoSuite);
+*/
```

Comment:
Disable two test suits? intentional?

Reply:
I think it's my fault. It's not intentional.
Thanks!